package.json

```diff
@@ -11,7 +11,7 @@
     "fxa-crypto-relier": "mozilla/fxa-crypto-relier#d45e814bc285ccdbc12783324407ae943915637f",
     "jose-jwe-jws": "^0.1.5",
     "kinto": "^10.0.0",
-    "kinto-http": "^4.3.4",
+    "kinto-http": "4.3.4",
```

Comment:
We shouldn't do that especially since we are fixing a bunch of bugs there at the moment.

Reply:
Ref: https://github.com/Kinto/kinto-http.js/issues/259 

Yea sorry, it was blocking people trying to build locally and try scoped keys. We should unlock it